pkgs/applications/office/scribus/unstable.nix

```diff
@@ -25,6 +25,11 @@ mkDerivation rec {
       url = "https://github.com/scribusproject/scribus/commit/b51c2bab4d57d685f96d427d6816bdd4ecfb4674.patch";
       sha256 = "031yy9ylzksczfnpcc4glfccz025sn47zg6fqqzjnqqrc16bgdlx";
     })
+    # fix build with Poppler 0.84
```

Comment:
Would be good to put a

```
# TODO: Remove with scribus version >= A.B.C
```

in case you know that it'll be released with the next version (e.g. when it's already in its `master`.

Reply:
Ok I added a comment; I hope that's good?